pennylane/math/multi_dispatch.py

```diff
@@ -340,3 +374,50 @@ def where(condition, x, y):
     tensor([ 0.6000,  0.2300,  0.7000, -4.0000, -5.0000], grad_fn=<SWhereBackward>)
     """
     return np.where(condition, x, y, like=_multi_dispatch([condition, x, y]))
+
+
+def unwrap(values, max_depth=None):
+    """Unwrap a sequence of objects to NumPy arrays.
+
+    Note that tensors on GPUs will automatically be copied
+    to the CPU.

```

Comment:
I wouldn't have thought of this, nice to be aware!

Reply:
I can't claim the credit for this unfortunately, `autoray.to_numpy()` has this built in 😆 

A nice artifact of using `autoray`, since I don't have a GPU and wouldn't have been able to test this myself.